src/createUiDefinition.json

```diff
@@ -205,6 +205,16 @@
                 "visible": "[equals(steps('clusterSettingsStep').azureCloudStorageAccountConfiguration.azureCloudPlugin, 'Yes')]"
               }
             ]
+          },
+          {
+            "name": "esAdditionalPlugins",
+            "type": "Microsoft.Common.TextBox",
+            "label": "Additional Elasticsearch Plugins",
+            "defaultValue": "",
+            "toolTip": "Additional elasticsearch plugins to install. Each plugin must be separated by a semicolon.  e.g. delete-by-query;lmenezes/elasticsearch-kopf/2.0",
+            "constraints": {
+              "required": false
+            }
```

Comment:
TBH I only added because I thought it was necessary to make the uiDefinition outputs work.
Will update the pull request.